src/PF1550/PF1550_IO.cpp

```diff
@@ -41,11 +41,11 @@ PF1550_IO::PF1550_IO(arduino::HardwareI2C * wire, uint8_t const i2c_addr)
    PUBLIC MEMBER FUNCTIONS
  ******************************************************************************/
 
-int PF1550_IO::begin()
+void PF1550_IO::begin()
 {
   _wire->begin();
```

Comment:
Hi @alrvid :coffee: :wave: 

Instead of eliminating the return value altogether I think it makes more sense to also return the return code of `_wire->begin();` and `_wire->setClock(100000);` in case there's an error. Can you please adjust this PR to that end?

Reply:
@aentinger Thanks. I think both `arduino::HardwareI2C` (and `TwoWire`) 's `begin()` and `setClock()` returns `void`.  So I made it `void`.

- https://github.com/arduino/ArduinoCore-API/blob/331cdd9ad216941cc2dedefa51fe643d1c4f0ac2/api/HardwareI2C.h#L29-L33
- https://github.com/arduino/ArduinoCore-avr/blob/63092126a406402022f943ac048fa195ed7e944b/libraries/Wire/src/Wire.h#L53-L57

Reply:
@aentinger What should I do about this one?